android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -314,15 +314,18 @@ public void addContact(ReadableMap contact, Callback callback) {
         ReadableArray phoneNumbers = contact.hasKey("phoneNumbers") ? contact.getArray("phoneNumbers") : null;
         int numOfPhones = 0;
         String[] phones = null;
-        Integer[] phonesLabels = null;
+        Integer[] phonesTypes = null;
+		String[] phonesLabels = null;
         if (phoneNumbers != null) {
             numOfPhones = phoneNumbers.size();
             phones = new String[numOfPhones];
-            phonesLabels = new Integer[numOfPhones];
+            phonesTypes = new Integer[numOfPhones];
+			phonesLabels = new String[numOfPhones];
```

Comment:
remove tab

Reply:
There is a lot more of unneeded tabbing, please check that :)

Reply:
sometimes this is the ide inserting stuff. I don't mind it either way. Just try to be consistent with the rest of the code @Git-Hub-Admin